geometry_msgs/Point[] points_to_check
---
bool is_occupied